### PR TITLE
Cleanup & consistency

### PR DESCRIPTION
**Pull request by BigMorty**
## Purpose

* Updated readme with cleaner steps
* Changed functions to consistent names (httpget & httppost)

## Does this introduce a breaking change?

```
[ ] Yes
[x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[x] Refactoring (no functional changes, no api changes)
[x] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
```

* Test the code

```
Use the test.http file
```

## What to Check
Verify that the following are valid
* Both httpget and httppost endpoints work

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,40 +51,36 @@ The project is designed to run on your local computer, provided you have met the
...
 
-3) Test a POST using your favorite REST client, e.g. [RestClient in VS Code](https://marketplace.visualstudio.com/items?itemName=humao.rest-client), PostMan, curl. `test.http` has been provided to run this quickly.
+3) Test the HTTP POST trigger using your favorite REST client (e.g. [RestClient in VS Code](https://marketplace.visualstudio.com/items?itemName=humao.rest-client)). `test.http` has been provided to run this quickly.
 Or in a new terminal run the following:
 
 ```bash
-curl -i -X POST http://localhost:7071/api/httppost \
-  -H "Content-Type: text/json" \
-  --data-binary  "@testdata.json"
+curl -i -X POST http://localhost:7071/api/httppost -H "Content-Type: text/json" -d "{\"name\": \"Developer\"}"
```
Comment (paulyuk) [inline README.md:68]: Works, but why not exact same object as the others (and escape in chars of course)?

Using this object:
```json
{
  "name": "Awesome Developer",
  "age": 25
}
```

The curl changes to

```suggestion
curl -i -X POST http://localhost:7071/api/httppost -H "Content-Type: text/json" -d "{\"name\": \"Awesome Developer\", \"age\": \"25\"}"
```

File: http/httpPostBodyFunction.cs [hunk 2]
```diff
@@ -20,15 +20,13 @@ public HttpPostBody(ILoggerFactory loggerFactory)
...
+            var returnValue = string.IsNullOrEmpty(person.Name)
+                ? "Hello, World."
+                : $"Hello, {person.Name}.";
             
             _logger.LogInformation($"C# HTTP trigger function processed a request for {person.Name}!");
-            return new OkObjectResult(person);
+            return new OkObjectResult(returnValue);
         }
     }
-    public record Person([property: JsonPropertyName("name")] string Name, [property: JsonPropertyName("age")] int Age);
```
Comment (paulyuk) [inline http/httpPostBodyFunction.cs:33]: Why change object ?

File: http/test.http [hunk 3]
```diff
@@ -2,10 +2,13 @@ GET http://localhost:7071/api/httpget HTTP/1.1
...
+GET http://localhost:7071/api/httpget?name=Developer HTTP/1.1
+
+###
+
 POST http://localhost:7071/api/httppost HTTP/1.1
 content-type: application/json
 
 {
-    "name": "Awesome Developer",
-    "age": 25
```
Comment (paulyuk) [inline http/test.http:10]: Same as above, liked the name and age Person object.  not sure why changing.  then the rest need to change too and not sure if better.

**Review by paulyuk — CHANGES_REQUESTED** [review]
Some comments inline

**Review by paulyuk — APPROVED** [review]
LGTM

File: README.md (hunk 1 above)
Comment (BigMorty) [reply README.md]: fixed

**Pull request merged**